src/components/show-edit-pod/show-edit-pod.component.js

```diff
@@ -37,8 +40,16 @@ const ShowEditPod = ({
   title,
   hideDeleteButtonInViewMode = false,
   variant = "transparent",
+  inputRef,
```

Comment:
You don't need this prop, it will never be set as it's not a part of this component's interface